.github/workflows/dockerprune.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             if [ -n $SECRET ]; then HAS_SECRET='true' ; fi
             echo "HAS_SECRET_TOKEN=${HAS_SECRET}" >> $GITHUB_OUTPUT
         env:
-          SECRET: "${{ secrets.DOCKER_PRUNE_TOKEN }}"
```

Comment:
Please don't change this I have separate token scopes for it.

Reply:
also - any thoughts on _why_ have it's worth having token with different scopes? they're both repo-level secrets presumably, unless you're using them for something else? maybe i should be doing the same...

Reply:
Might have a couple typos in should check. I just like seperate tokens for different things. Also the DOCKER_TOKEN I generated is super old and dosent seem to have the right privileges to do what I want in that script. Would always throw a access denied until I generated that new token just for pruning.

Reply:
Switched back to `DOCKER_PRUNE_TOKEN` and fixed the typo